mfrc522.py

```diff
@@ -32,6 +32,9 @@ def __init__(self, sck, mosi, miso, rst, cs):
 		elif board == 'esp8266':
 			self.spi = SPI(baudrate=100000, polarity=0, phase=0, sck=self.sck, mosi=self.mosi, miso=self.miso)
 			self.spi.init()
+		elif board == 'esp32':
+			self.spi = SPI(baudrate=100000, polarity=0, phase=0, sck=self.sck, mosi=self.mosi, miso=self.miso)
+			self.spi.init()
```

Comment:
as you are doing precisely the same thing as 3 lines above, how about just changing line 32 to:
```
elif board == 'esp8266' or board == 'esp32':
```